apps/ejabberd/src/rdbms/mongoose_rdbms.erl

```diff
@@ -473,7 +473,7 @@ connect(Settings, Retry, RetryAfter, MaxRetryDelay) ->
                        " Retrying in ~p seconds.", [Settings, Error, SleepFor]),
             timer:sleep(timer:seconds(SleepFor)),
             NextRetryDelay = RetryAfter * RetryAfter,
-            connect(Settings, Retry - 1, max(MaxRetryDelay, NextRetryDelay), MaxRetryDelay)
+            connect(Settings, Retry - 1, min(MaxRetryDelay, NextRetryDelay), MaxRetryDelay)
```

Comment:
The coverage you can fine here: https://coveralls.io/builds/12357642/source?filename=apps%2Fejabberd%2Fsrc%2Frdbms%2Fmongoose_rdbms.erl#L476